Buildings/Fluid/Humidifiers/Validation/DXDehumidifier.mo

```diff
@@ -0,0 +1,233 @@
+    final tPer=3600,
+    final tDel=0.1)
+    "Convert PLR signal to on-off signal"
+    annotation (Placement(transformation(extent={{-120,60},{-100,80}})));
+
+  Buildings.Fluid.Sources.MassFlowSource_T boundary(
+    redeclare package Medium = Medium,
+    final use_Xi_in=true,
+    final use_m_flow_in=true,
+    final use_T_in=true,
+    nPorts=1)
+    "Mass flow source for coil inlet air"
+    annotation (Placement(transformation(extent={{-82,-28},{-62,-8}})));
+
+  Buildings.Fluid.Sources.Boundary_pT sin(
+    redeclare package Medium = Medium,
+    final p(displayUnit="Pa") = 101325,
+    final T=294.15,
+    nPorts=1)
+    "Sink"
+    annotation (Placement(transformation(extent={{0,-28},{-20,-8}})));
+
+  Buildings.Utilities.Psychrometrics.ToTotalAir toTotAirIn
+    "Convert humidity ratio per kg dry air to humidity ratio per kg total air for coil inlet air"
+    annotation (Placement(transformation(extent={{-120,-60},{-100,-40}})));
+
+  Buildings.Utilities.IO.BCVTB.From_degC TIn_K
+    "Converts degC to K"
+    annotation (Placement(transformation(extent={{-120,10},{-100,30}})));
+
+  Buildings.Controls.OBC.CDL.Conversions.BooleanToReal booToReaFanEna
+    "Convert fan enable signal to real value"
+    annotation (Placement(transformation(extent={{-160,-20},{-140,0}})));
+
+  Buildings.Controls.OBC.CDL.Reals.MultiplyByParameter gai(
+    final k=m_flow_nominal)
+    "Calculate inlet air mass flow rate from component enable signal"
+    annotation (Placement(transformation(extent={{-120,-20},{-100,0}})));
+
+  Modelica.Blocks.Sources.RealExpression realExpression1(
+    final y=dxDeh.port_a.m_flow)
+    "DX dehumidifier air mass flow rate (Modelica)"
+    annotation (Placement(transformation(extent={{20,60},{40,80}})));
+
+  Modelica.Blocks.Math.Mean m_flowFan(
+    final f=1/tStepAve)
+    "Average out Modelica results over time"
+    annotation (Placement(transformation(extent={{54,60},{74,80}})));
+
+  Modelica.Blocks.Math.Mean m_flowFanEP(
+    final f=1/tStepAve)
+    "Average out EnergyPlus results over time"
+    annotation (Placement(transformation(extent={{134,60},{154,80}})));
+
+  Modelica.Blocks.Sources.RealExpression realExpression5(
+    final y=datRea.y[6])
+    "DX dehumidifier air mass flow rate (EnergyPlus)"
+    annotation (Placement(transformation(extent={{98,60},{118,80}})));
+
+  Modelica.Blocks.Sources.RealExpression realExpression2(
+    final y=-dxDeh.deHum.mWat_flow)
+    "Water removal mass flow rate (Modelica)"
+    annotation (Placement(transformation(extent={{20,26},{40,46}})));
+
+  Modelica.Blocks.Math.Mean mWatMod(
+    final f=1/tStepAve)
+    "Average out Modelica results over time"
+    annotation (Placement(transformation(extent={{54,26},{74,46}})));
+
+  Modelica.Blocks.Math.Mean mWatEP(
+    final f=1/tStepAve)
+    "Average out EnergyPlus results over time"
+    annotation (Placement(transformation(extent={{134,26},{154,46}})));
+
+  Modelica.Blocks.Sources.RealExpression realExpression6(
```

Comment:
change the instance name to be like `watRemRat`, so we now what the instance is doing.